src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -248,8 +249,12 @@ def _copy_previous_run_result(
                     # Thus the root_run_id needs to be the current batch run id.
                     previous_run_info.root_run_id = run_id
                     previous_run_info.parent_run_id = run_id
-                    # Load previous node run info
+
+                    # Load previous node run info and remove aggregation nodes in case it is loaded into node run info
```

Comment:
Please add more comment to explain why aggregation node run info would be loaded in previous_node_run_infos
Just describe your findings as it is